client.go

```diff
@@ -54,22 +57,27 @@ func (c *Client) Login() error {
 		return err
 	}
 	c.token = data.Token
+
+	go c.refreshToken()
 	return nil
 }
 
-// RefreshToken is used to refresh the current token.
-func (c *Client) RefreshToken() error {
-	resp, err := c.performGETRequest("/refresh_token", nil)
-	if err != nil {
-		return err
-	}
-	defer resp.Body.Close()
-	data := new(loginAPIResponse)
-	err = parseResponse(resp.Body, &data)
-	if err != nil {
-		return err
+// refreshToken is used to refresh the current token.
+func (c *Client) refreshToken() error {
+	for range tickCh {
+		resp, err := c.performGETRequest("/refresh_token", nil)
+		if err != nil {
+			return err
+		}
+		defer resp.Body.Close()
+		data := new(loginAPIResponse)
+		err = parseResponse(resp.Body, &data)
+		if err != nil {
+			return err
+		}
+		c.token = data.Token
```

Comment:
May be we need to add a lock to prevent race conditions because `RefreshToken` is public?